flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/model/SequenceNumber.java

```diff
@@ -0,0 +1,87 @@
+package org.apache.flink.streaming.connectors.kinesis.model;
```

Comment:
Yes, I used verify. The problem was that I was building from `flink/flink-streaming-connectors` instead of root to save some time, and the RAT plugin wasn't executed. I'll remember next time ;)